packages/vats/tools/fake-bridge.js

```diff
@@ -216,3 +217,47 @@ export const makeFakeLocalchainBridge = (zone, onToBridge = () => {}) => {
     },
   });
 };
+
+/**
+ * @param {import('@agoric/zone').Zone} zone
+ * @param {(obj) => void} [onToBridge]
+ * @returns {ScopedBridgeManager<'vtransfer'>}
+ */
+export const makeFakeTransferBridge = (zone, onToBridge = () => {}) => {
+  /** @type {Remote<BridgeHandler>} */
+  let hndlr;
+  const registered = zone.setStore('registered');
```

Comment:
I'm undecided about whether or not this is an abuse of the provided `zone`, but leaning towards not. However, it _is_ less discoverable than passing in an argument with the right `add`/`delete` interface.

Reply:
do you anticipate the caller needing access to the set? if not, this lgtm. It matches the other makeFake…Bridge function signatures (all starting with `zone`)

Reply:
> do you anticipate the caller needing access to the set? if not, this lgtm. It matches the other makeFake…Bridge function signatures (all starting with `zone`)

Accepting `(zone, …)` makes sense; the part I'm on the fence about is putting a new set in that zone rather than having the set be a simple ephemeral `new Set()` (analogous to local variable `lcaExecuteTxSequence` in `makeFakeLocalchainBridge`) or accepting it as a separate argument, whereas this way implies that it is part of some unspecified API. But I don't consider it a blocking issue.